lib/dotcom/system_status/groups.ex

```diff
@@ -0,0 +1,221 @@
+defmodule Dotcom.SystemStatus.Groups do
+  @moduledoc """
+  A module that groups alerts into statuses for the system status widget.
+  """
+
+  alias Alerts.Alert
+
+  @lines ["Blue", "Orange", "Red", "Green"]
```

Comment:
Is there a reason to this order? If not, we should alphabetize.

Reply:
This is the order that routes will get sorted in on the widget when they get rendered (I got this order from Figma, but we also discussed changing it to `["Red", "Orange", "Green", "Blue"]` to match the GTFS `route_sort_order`, or `["Red", "Orange", "Blue", "Green"]` to be ordered first by heavy rail before light rail, and then by `route_sort_order`).

I'll add an explanatory comment and sort the other ones.